models/spleen_deepedit_annotation/docs/README.md

```diff
@@ -21,6 +21,34 @@ Input: 3 channels CT image - one channel representing clicks for each segment (i
 
 Output: 2 channels: Label 1: spleen; Label 0: everything else - This depends on the dictionary "label_names" defined in train.json and inference.json ("label_names": {"spleen": 1, "background": 0},)
 
+## Model Performance
+
+The achieved Dice score on the validation set is:
+
+Spleen: 0.971
```

Comment:
Hi @diazandr3s , thanks for the updates, I still cannot reproduce this number. The following is the log:
```
2022-11-14 03:17:04,180 - INFO - --- input summary of monai.bundle.scripts.run ---
2022-11-14 03:17:04,180 - INFO - > runner_id: 'evaluating'
2022-11-14 03:17:04,180 - INFO - > meta_file: 'configs/metadata.json'
2022-11-14 03:17:04,180 - INFO - > config_file: ['configs/train.json', 'configs/evaluate.json']
2022-11-14 03:17:04,180 - INFO - > logging_file: 'configs/logging.conf'
2022-11-14 03:17:04,180 - INFO - > bundle_root: '.'
2022-11-14 03:17:04,181 - INFO - > dataset_dir: '/workspace/data/medical/Task09_Spleen/'
2022-11-14 03:17:04,181 - INFO - ---


2022-11-14 03:17:04,181 - INFO - set logging properties based on config: configs/logging.conf.
there are more than 1 component have name `Interaction`: ['monai.apps.deepedit.interaction', 'monai.apps.deepgrow.interaction'], use the first one `monai.apps.deepedit.interaction. if want to use others, please set its full module path in `_target_` direc
tly.
2022-11-14 03:17:05,428 - ignite.engine.engine.SupervisedEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2022-11-14 03:17:05,501 - ignite.engine.engine.SupervisedEvaluator - INFO - Restored all variables from ./models/model.pt
2022-11-14 03:17:10,964 - INFO - Number of simulated clicks: 5
2022-11-14 03:17:12,654 - INFO - Number of simulated clicks: 6
2022-11-14 03:17:15,416 - INFO - Number of simulated clicks: 2
2022-11-14 03:17:17,321 - INFO - Number of simulated clicks: 5
2022-11-14 03:17:19,385 - INFO - Number of simulated clicks: 5
2022-11-14 03:17:21,137 - INFO - Number of simulated clicks: 5
2022-11-14 03:17:22,800 - INFO - Number of simulated clicks: 5
2022-11-14 03:17:24,256 - INFO - Number of simulated clicks: 3
2022-11-14 03:17:25,732 - INFO - Number of simulated clicks: 3
2022-11-14 03:17:26,005 - ignite.engine.engine.SupervisedEvaluator - INFO - Got new best metric of val_mean_dice: 0.9151045680046082
2022-11-14 03:17:26,006 - ignite.engine.engine.SupervisedEvaluator - INFO - Epoch[1] Metrics -- val_mean_dice: 0.9151
2022-11-14 03:17:26,006 - ignite.engine.engine.SupervisedEvaluator - INFO - Key metric: val_mean_dice best value: 0.9151045680046082 at epoch: 1
Mean of empty slice
All-NaN slice encountered
All-NaN slice encountered
All-NaN slice encountered
Degrees of freedom <= 0 for slice.
2022-11-14 03:17:26,010 - ignite.engine.engine.SupervisedEvaluator - INFO - Epoch[1] Complete. Time taken: 00:00:20.508
2022-11-14 03:17:26,010 - ignite.engine.engine.SupervisedEvaluator - INFO - Engine run complete. Time taken: 00:00:20.582
[null, null]
```
It got 0.9151 mean dice. In addition, I run the evaluating command for 3 times, and produced 3 different mean dice scores. I'm not familiar with the deepedit method, is it correct that the deepedit method cannot produce deterministic results?

Reply:
This depends on the number of simulated clicks. DeepEdit is both an automatic and interactive method. It performs better if the user provides clicks or more clicks are simulated during validation.

Reply:
I added the reference to the DeepEdit paper so users know more about this method.

Reply:
thanks for the explain, just to make sure if the weights file is align with the readme file.
In the user point of view (as what I checked here), they may care about how to make use of the weights and reach to 0.971. Could you please help to add more descriptions about how to reach to this number (like provide clicks as you said)? and if can not get the deterministic result, may need to write sth like `around 0.971`?